frame/support/test/tests/pallet.rs

```diff
@@ -1590,6 +1590,30 @@ fn metadata() {
 	pretty_assertions::assert_eq!(actual_metadata.pallets, expected_metadata.pallets);
 }
 
+#[test]
+fn metadata_at_version() {
+	use frame_support::metadata::*;
+	use sp_core::Decode;
+
+	let actual_metadata = Runtime::metadata();
```

Comment:
Nit; maybe `let v14_metadata`? Since that's all this call will ever return :)

Reply:
Have changed this to `let metadata` since now we can control the version via the `LATEST_METADATA_VERSION` test constant, thanks!